doc/dune-files.rst

```diff
@@ -761,6 +761,17 @@ fields. In particular, all fields except for ``public_names`` are supported from
 the :ref:`executables stanza <shared-exe-fields>`. Alias fields apart from
 ``name`` are allowed.
 
+The ``action`` field can be used to override the test binary invocation, for example
```

Comment:
Can you describe the default? (`%{test}` is run without argument)

Reply:
Sure, I'll add that!